src/client/components/vision/grid_layout/grid_layout.tsx

```diff
@@ -44,7 +44,7 @@ export class GridLayout extends Component<{
   @computed
   get gridTemplate() {
     const n = React.Children.count(this.props.children)
-    if (!this.width || !this.height) {
+    if (!this.width || !this.height || n < 2) {
```

Comment:
Returns `1fr`(single row, single column) when count of children is zero or one. There's no need to call `bestFitCols()` in this case.